### ISSUE TITLE
Socks5 Proxy

### ISSUE THREAD

**Issue by jalali110** [Liupold/ODM#1]
Hi there,
It would be very useful if the downloader also supports Socks5 Proxy.

**Comment by Liupold** [comment]
Yes it would be, an example can be found [here](https://stackoverflow.com/questions/12601316/how-to-make-python-requests-work-via-socks-proxy).

I am not sure if I get it implemented soon. I would be more than happy to merge the pull request if you can get this implemented.
